### PR TITLE
XW-113 | Fix for backgrounds modals

### PR DESCRIPTION
**Pull request by vforge**
/cc @bognix @Warkot

### CODE REVIEW

File: resources/wikia/ui_components/modal/css/modal_default.scss [hunk 1]
```diff
@@ -227,12 +224,13 @@ body.with-blackout {
...
 
 	.global-navigation {
-		position: static;
+		position: fixed;
 	}
 
 	.WikiaSiteWrapper {
 		position: fixed;
 		width: 100%;
+		z-index: 2;
```
Comment (Warkot) [inline resources/wikia/ui_components/modal/css/modal_default.scss:233]: Don't we have `z-index` variables in our `SCSS`?

Comment (vforge) [inline resources/wikia/ui_components/modal/css/modal_default.scss:233]: Not for this, see https://github.com/Wikia/app/pull/8227/files#diff-01b6b4c2965acf4b57d1e418b901c041R163

Comment (Warkot) [inline resources/wikia/ui_components/modal/css/modal_default.scss:233]: We could improve this, but that's a different story… :books:

File: resources/wikia/ui_components/modal/css/modal_default.scss [hunk 2]
```diff
@@ -209,10 +209,7 @@ $large-modal-margin: 50px;
 
 // styles required to disable scrolling behind modal
 body.with-blackout {
-
 	&.background-dynamic {
-		position: static;
```
Comment (Warkot) [inline resources/wikia/ui_components/modal/css/modal_default.scss:214]: `&.background-`**`dynamic`** {
`position:`**`static`**`;`

That didn't sound to dynamic to me. :grin:

**Comment by Warkot** [comment]
LGTM :+1:

**Comment by bognix** [comment]
LGTM

**Pull request merged**